components/connectivity-adapter/internal/appregistry/service/graphql.go

```diff
@@ -0,0 +1,16 @@
+package service
```

Comment:
I would expect this file will grow during the implementation, it would be valuable to create a separate package for it.

Reply:
Let's wait with this decision until this file grows, okay? Because my idea was to have only GraphQL requests here, and for the app registry it shouldn't be much. Let's see what the future brings 🙂